packages/desktop-client/src/components/sidebar/SecondaryItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {
+  type CSSProperties,
+  type ComponentType,
+  type MouseEventHandler,
+  type SVGProps,
+} from 'react';
+
+// eslint-disable-next-line no-restricted-imports
+import { styles, colors } from '../../style';
+import Block from '../common/Block';
+import View from '../common/View';
+
+import { ItemContent } from './ItemContent';
+
+const fontWeight = 600;
+
+let accountNameStyle = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we could import this from the `Account` component file to reduce duplication.